### DISCUSSION TITLE
1.4.2.3 Release

### DISCUSSION THREAD

**Discussion by Sodrrr** [Pryaxis/TShock#2321] [Q&A] [answered]
When will there be a release for 1.4.2.3?

**Comment by ghost** [comment]
when it.s going to be done

**Reply by Aro2220** [reply]
excuse my vapid friend we are all just excited for your work as always

**Comment by Arthri** [comment, marked as answer]
You can try out an experimental build for now, this is a tutorial on how to get one by @hakusaro 

https://user-images.githubusercontent.com/41360489/119002730-b581b200-b9bf-11eb-910e-69da9cdb0964.mp4
